src/mslice/plotting/plot_window/slice_plot.py

```diff
@@ -609,3 +634,24 @@ def _get_overplot_datum():  # needed for interface consistency with cut plot
 
     def set_cross_cursor(self):
         self._canvas.setCursor(Qt.CrossCursor)
+
+    @property
+    def all_fonts_size(self):
+        font_sizes_config = {}
+        for p in self.plot_fonts_properties:
+            font_sizes_config[p] = getattr(self, p)
+
+        return font_sizes_config
+
+    @all_fonts_size.setter
+    def all_fonts_size(self, values: dict):
+        for key in values:
+            setattr(self, key, values[key])
+
+    def increase_all_fonts(self):
+        for p in self.plot_fonts_properties:
+            setattr(self, p, getattr(self, p) + 0.5)
```

Comment:
Incrementing and decrementing font sizes works really well, so good job with this. The only thing I would change is I think 0.5 is quite small, perhaps we want it to be 1 to keep the font sizes at whole numbers?

It would also be good to have this number as a variable at the top of the file in capitals to signify it is a constant